spring-cloud-dataflow-registry/src/main/java/org/springframework/cloud/dataflow/registry/support/AppResourceCommon.java

```diff
@@ -34,10 +35,66 @@
 import org.springframework.util.StringUtils;
 
 /**
- * @author Mark Pollack
- * @author Ilayaperumal Gopinathan
+ * @author Christian Tzolov
  */
-public class ResourceUtils {
+public class AppResourceCommon {
+
+	/**
+	 * the maven properties to use in case of maven resource
+	 */
+	private MavenProperties mavenProperties;
+
+	/**
+	 *
```

Comment:
Maybe just remove empty doc block

Reply:
Ok will try to infer some description

Reply:
also it seems that `metadataResourceLoader` is better name for this field